packages/canary/src/styles.css

```diff
@@ -195,3 +195,11 @@
 .entity-list-action-chevron {
   @apply transition-transform transform rotate-0 text-tertiary-background group-hover:text-primary;
 }
+
+/* Temporary studio controls override */
+.react-flow__panel {
```

Comment:
these should not be in canary package, as canary should not be aware of pipeline studio.
we can add them in playground styles.css

Reply:
This was a short term fix for the demo but you are absolutely correct.